### DISCUSSION TITLE
.env file not available in project then how to show the value in process.env.SHOPIFY_API_KEY

### DISCUSSION THREAD

**Discussion by chiragDesaiPantherCodx** [Shopify/shopify-app-template-node#1007] [Q&A]
Shopify app in template selected node, No env file available in structure but process.env.SHOPIFY_API_KEY value access them.
How to set value in  process.env.SHOPIFY_API_KEY and how to get value without .env file.

**Comment by capsloq** [comment]
env variables do get "magically" pulled from shopify partners by the shopify cli.
You can pull them (this will create a .env in your project root)
`yarn shopify app env pull`
or view them
`yarn shopify app env show`

Documentation here:
https://shopify.dev/apps/tools/cli/commands#env
